src/tables.jl

```diff
@@ -38,10 +38,11 @@ function Tables.columns(A::Union{KeyedArray, NdaKa})
     L = hasnames(A) ? (dimnames(A)..., :value) :
         (ntuple(d -> Symbol(:dim_,d), ndims(A))..., :value)
     R = keys_or_axes(A)
-    G = ntuple(ndims(A)) do d
-        vec([rs[d] for rs in Iterators.product(R...)])
-        # _vec(rs[d] for rs in Iterators.product(R...))
-    end
+    R_inds = map(eachindex, R)
+    G = map(
+        (r, d) -> vec([r[rs[d]] for rs in Iterators.product(R_inds...)]),
+        R, ntuple(identity, length(R)),
```

Comment:
Isn't this the same as `tuple(eachindex(R_inds))`?

Reply:
Hmmm I'm not sure. If `length(R)` is `2`, then
```julia
ntuple(identity, length(R))
```
just produces the `Tuple`
```julia
(1, 2)
```

Reply:
This doesn't feel like an elegant way to produce this particular tuple though, I agree

Reply:
Isn't that what this is doing?

```
julia> ntuple(identity, 2)
(1, 2)
```

Reply:
Hmm I'm confused haha

> Isn't this the same as tuple(eachindex(R_inds))?

Exactly which bit of code were you referring to with this comment?

Reply:
Just cause calling `ntuple` with `identity` seems a bit confusing, so something like

```suggestion
        R, tuple(eachindex(R_inds)),
```

might be a bit clearer. AFAICT, there's no performance hit.